airflow/providers/google/cloud/links/dataproc.py

```diff
@@ -32,6 +32,8 @@
 
 DATAPROC_BASE_LINK = BASE_LINK + "/dataproc"
 DATAPROC_JOB_LINK = DATAPROC_BASE_LINK + "/jobs/{job_id}?region={region}&project={project_id}"
+# For backward-compatibility, DATAPROC_JOB_LINK was DATAPROC_JOB_LOG_LINK.
+DATAPROC_JOB_LOG_LINK = DATAPROC_JOB_LINK
```

Comment:
Lets please add TODO of how/what to change when we plan a major release

Reply:
Not sure if we want to do something. This is 0 -maintenance, 0 effort to keep it forever. I think we should consider removal of some features (that we consider as Public API) only when it actually costs us anything.

Reply:
Regardless when we will do it we need to have the instructions of what to do.

Reply:
Agree

Reply:
Sure. Just added.